### DISCUSSION TITLE
Add arabic language to CP

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#6249] [Q&A]
I want to add arabic laguage to CP so the admin can switch between english and arabic. Please support ASAP

**Comment by jackmcdade** [comment]
As soon as someone contributes the translation it’ll be available. All translations are community supported.

**Reply by duncanmcclean** [reply]
On the back of this - there's documentation if you wish to [contribute Arabic as a language](https://statamic.dev/cp-translations#contributing-a-new-translation).
